function-logs/handler.go

```diff
@@ -1,20 +1,34 @@
 package function
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"net/http"
 	"net/url"
 	"os"
 	"strings"
 	"time"
 
+	faasSDK "github.com/openfaas/faas-cli/proxy"
+	"github.com/openfaas/faas-provider/logs"
+	"github.com/openfaas/faas-provider/types"
 	"github.com/openfaas/openfaas-cloud/sdk"
 )
 
+type FaaSAuth struct{}
+
+func (auth *FaaSAuth) Set(req *http.Request) error {
+	return sdk.AddBasicAuth(req)
+}
+
+var (
+	timeout   = 3 * time.Second
```

Comment:
This may not be enough for when the system is under contention, perhaps 5s might be a safer bet?

Reply:
Increased to 5 :+1: